### ISSUE TITLE
imageMagic blocked

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#486]
here the next issue running GGI on this machine

![image](https://user-images.githubusercontent.com/4609956/44384117-5b581780-a51b-11e8-8686-465f69e8a96f.png)

**Comment by myrmoteras** [comment]
solved by making an exception for imageMagic in bitdefender

**Comment by gsautter** [comment]
Yeah, that's but the only hint I could have given you anyway ... system protection software is no joke. I'm afraid we'll keep seeing these kinds of problems for a bit. Preciously little we could do about it as system protection tools grow increasingly paranoid.
